{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -329,6 +329,8 @@
 CELERY_TASK_SEND_SENT_EVENT = True
 # https://docs.celeryq.dev/en/stable/userguide/configuration.html#worker-hijack-root-logger
 CELERY_WORKER_HIJACK_ROOT_LOGGER = False
+# https://docs.celeryq.dev/en/stable/userguide/configuration.html#broker-connection-retry-on-startup
+CELERY_BROKER_CONNECTION_RETRY_ON_STARTUP = True
```

Comment:
Can we move this closer to the others `CELERY_BROKER_...` settings please and set it to `False`:

https://github.com/cookiecutter/cookiecutter-django/issues/5085#issuecomment-2211735674